kalite/control_panel/views.py

```diff
@@ -558,6 +560,27 @@ def _get_user_usage_data(users, groups=None, period_start=None, period_end=None,
     return (user_data, group_data)
 
 
+def check_meta_data(facility_id):
+    '''Checks whether any metadata is missing for the specified facility.
+
+    Args: 
+      facility_id (str): id of facility to check
+ 
+    Returns:
+      bool: True if one or more metadata fields are missing'''
+
+    facility_data = {}
+    facility_entry = Facility.objects.filter(id=facility_id)[0]
+    facility_data['usercount'] = facility_entry.user_count
```

Comment:
This can be collapsed to two lines:

```
check_fields = ['usercount', 'latitude', 'longitude', 'address', 'contact_name', 'contact_phone', 'contact_email']
return any([getattr(facility, value, None) is None for value in check_fields])
```